### ISSUE TITLE
Work on HD torrent or ipt ?

### ISSUE THREAD

**Issue by ruCyberPoison** [anthonyraymond/joal#61]
Hello i have tested your app and looks like working on public torrent and on semi private but in private torrents like HDT or IPT the app doesn't work, the log says 0 leeacher when in reallity we have 10 or more leachers to seed.

Can you fix it ?

**Comment by anthonyraymond** [comment]
I'm pretty confident that it work on IPT.
If 0 leechers appear it's because IPT sayd that there are none. It's possible that you still see some seeders / leechers on your real bittorrent client if you have enable peer exchange or DHT though

**Comment by anthonyraymond** [comment]
Closing due to no further answer, feel fre to reopen or add a reply if you still have any issues

**Comment by ruCyberPoison** [comment]
I tested on HD Torrent and he doesn't seed noting just close the torrent or remove automatically can you check from your side to be sure than this is not a mistake from my part. 

On IPT works great, just in HD Torrent doesn't work as expected. 

Rgds

Sorry to reply only after a week ago

**Comment by anthonyraymond** [comment]
I don't have an account on HDTorrent, can you send me the console log from JOAL?
**Be sure to hide your IP / passkey from the log before sending.**

Also, have you checked the `keep torrent with zero peers` option?

**Comment by anthonyraymond** [comment]
Closing due to no answer

**Comment by deadpansociety** [comment]
crazyhd does the same thing like HD-T. (message me your email for an invite)

Here's a portion of HD-T log

```
[INFO ] 2019-04-15 14:47:00.019 [   annnouncer-1] o.a.j.c.t.c.a.Announcer: ---deleted--- has announced successfully. Response: 0 seeders, 0 leechers, 1800s interval
[INFO ] 2019-04-15 14:47:00.035 [MessageBroker-1] o.s.w.s.c.WebSocketMessageBrokerStats: WebSocketSession[1 current WS(1)-HttpStream(0)-HttpPoll(0), 1 total, 0 closed abnormally (0 connect failure, 0 send limit, 0 transport error)], stompSubProtocol[processed CONNECT(1)-CONNECTED(1)-DISCONNECT(0)], stompBrokerRelay[null], inboundChannel[pool size = 8, active threads = 0, queued tasks = 0, completed tasks = 48], outboundChannelpool size = 8, active threads = 0, queued tasks = 0, completed tasks = 10], sockJsScheduler[pool size = 1, active threads = 1, queued tasks = 0, completed
tasks = 0]
```
(also if I've written my passkey above please notify)

To be fair, neither Ratiomaster can show correct seed/leech number, it's also 0/0 there.
Keep torr. w/ 0 peers is checked.

**Comment by anthonyraymond** [comment]
It may be possible that hdt does only procide peers through scrape request, which joal does not handles

**Comment by chicungunya** [comment]
Sorry if i create a duplicate of this issue, but same problem for me.
https://github.com/anthonyraymond/joal/issues/147
